### ISSUE TITLE
feat: integration with [ui.use.ink](https://ui.use.ink)

### ISSUE THREAD

**Issue by brunopgalvao** [r0gue-io/pop-cli#292]
Currently when I deploy using Pop CLI, the deployed contract does not show up in [ui.use.ink](https://ui.use.ink) (or PolkadotJs Apps > Contracts). 

As a developer, I need a way for when I deploy a contract using Pop CLI, that it shows up on the [ui.use.ink](https://ui.use.ink) because that UI is useful for debugging and part of the development process.

**Comment by AlexD10S** [comment]
Are you experimenting the same when deploying a smart contract using `cargo-contract?`
This might need more research but seems more an issue of ui.use.ink that fails to show the contract.

**Comment by brunopgalvao** [comment]
> Are you experimenting the same when deploying a smart contract using `cargo-contract?` This might need more research but seems more an issue of ui.use.ink that fails to show the contract.

Same. For both cases I have to go to ui.use.ink and manually add the code_hash of the deployed contract for the contract to show up in the UI.

**Comment by AlexD10S** [comment]
It seems like the issue might be on the `ui.use.ink` side, right? If the contract is deployed correctly on-chain, there’s probably not much we can address directly from the CLI.

**Comment by brunopgalvao** [comment]
> It seems like the issue might be on the `ui.use.ink` side, right? If the contract is deployed correctly on-chain, there’s probably not much we can address directly from the CLI.

Yes, and it is critical to the success of Pop CLI smart contract development workflow.

**Comment by Daanvdplas** [comment]
I want to bring up this issue because I think indeed it is a very good one to polish the developer journey @AlexD10S (thanks @brunopgalvao ❤️)

**Comment by AlexD10S** [comment]
This is a significant feature and will require integration similar to what we did with PDP. We'll need a way to send metadata and the address (or hash) to the ui.use.ink API so it can be displayed on the website.
Currently, ui.use.ink doesn't have an API or backend server in place.
